docs/tests.md

```diff
@@ -0,0 +1,18 @@
+# Tests for actors
+
+You can implement two types of tests for an actor in the leapp-repository - functional and unit tests.
+
+## Functional tests
```

Comment:
From @AloisMahdal: do not use word _functional_. All tests we'll have for Leapp will be functional. Non-functional tests are stress tests, performance tests, load tests, etc. Use _component_ instead.

Reply:
I googled E2E testing some time ago and generally the term seems to be defined in a very high-level way, meaning something along lines of: the process of carrying out very thorough and complete set of testing activities.

(Even the way we use it in OAMG-QE is sometimes a bit too specific; sometimes we use it just to refer to "automated upgrade tests".)

So I would prefer *component test*.

Reply:
Agree with Lojza. Component test fits better to me.

Reply:
I am voting here for the name component test as well, it seems to be clear, plus, we are considering actor as a component of the whole system. Having the name E2E here would require more distinguishing and deeper specification. Personally, I do not want to go that way as there are other things to solve than definition of the words used here.

Reply:
_component_ it is then..